plugins/runner-ts/src/index.ts

```diff
@@ -95,47 +96,49 @@ export default defineRunner(tsRunnerOptions, async ctx => {
     getCurrentDirectory: () => process.cwd(),
     getDefaultLibFileName: options => ts.getDefaultLibFilePath(options),
     fileExists: fileName => {
-      if (filesCache[fileName]) {
+      if (filesCache[Path.normalize(fileName)]) {
```

Comment:
so, Path.normalize is enough? No need for the normalize-path?

Reply:
From what I tested, Path.normalize seems to be correct, it's a bit confusing to me, because Path.normalize takes a path, and makes it platform-specific, whereas normalize-path takes a path, and conforms it to Unix (hence we can use them in jest tests on Windows to get a Unix path), yet they both suggest that they "normalize" the path.

Reply:
ah yeah, you're right